build.cake

```diff
@@ -125,6 +125,7 @@ Task("Run-Unit-Tests")
                 parameters.Paths.Files.TestCoverageOutputFilePath,
                 new OpenCoverSettings {
                     ReturnTargetCodeOffset = 0,
+                    Register = "user",
```

Comment:
@devlead ah, i see :-(

That would be one approach, yes.  However, now that OpenCover wraps the Test Runner, skipping OpenCover would also skip the Unit Testing, which is not ideal.  We would need to structure the task differently to allow running of the tests directly, when that environment variable is not set.